modules/workload-identity/variables.tf

```
@@ -101,3 +101,9 @@ variable "use_existing_context" {
   type        = bool
   default     = false
 }
+
+variable "module_depends_on" {
+  description = "List of modules or resources this module depends on."
```

Since this will only control the gcloud invocation.
```suggestion
  description = "List of modules or resources to depend on before annotating KSA."
```